tasks/source_artifactory.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Get corda jars from Artifactory (for SNAPSHOTS)
+  get_url:
+    url: 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-dev/net/corda/corda/{{ corda_version }}/corda-{{ corda_version }}.jar'
```

Comment:
`corda-dev` cannot be hardcoded. It is unlikely a Corda release will be in that snapshot repository

Reply:
Technically, speaking you are right. However, for non-snapshots maven should be sources of packages. This is just for people will to test incoming release.